cmd/lagoon-opensearch-sync/sync.go

```diff
@@ -23,6 +23,7 @@ type SyncCmd struct {
 	Period                      time.Duration `kong:"default='8m',help='Period between synchronisation polls'"`
 	Objects                     []string      `kong:"enum='tenants,roles,rolesmapping,indexpatterns,indextemplates',default='tenants,roles,rolesmapping,indexpatterns,indextemplates',help='Opensearch objects which will be synchronized'"`
 	LegacyIndexPatternDelimiter bool          `kong:"default='false',help='Use the legacy -* index pattern delimiter instead of -_-*'"`
+	HTTPClientTimeout           time.Duration `kong:"default='30s',env='HTTP_CLIENT_TIMEOUT',help='HTTP client timeout for API requests'"`
```

Comment:
Same comment about the naming here, and it illustrates the issue with the `HTTP` naming.

This value is passed to two different clients below which are talking to completely different APIs. So I think we need two options here:

* `OpensearchClientTimeout`
* `OpensearchDashboardsClientTimeout`